src/components/UNSTABLE__LayoutModules/Layout.stories.js

```diff
@@ -0,0 +1,567 @@
+/**
```

Comment:
This file should now be split out into `Detail.stories` and `Overview.stories` and should no longer be necessary